recipe/meta.yaml

```diff
@@ -21,14 +21,16 @@ requirements:
   build:
     - {{ compiler("c") }}
     - {{ compiler("cxx") }}
-    - {{ cdt("libnl3-devel") }}
+    - libnl
```

Comment:
Think we can drop this from `build` now that it is in `host`

CDTs are a bit special as they get added to `build`, but most cases dependencies that are linked against are added to `host`

```suggestion
```